addons/mail/static/src/messaging/entity/notification/notification.js

```diff
@@ -0,0 +1,69 @@
+odoo.define('mail.messaging.entity.Notification', function (require) {
+'use strict';
+
+const { registerNewEntity } = require('mail.messaging.entityCore');
+const { attr, many2one } = require('mail.messaging.EntityField');
+
+function NotificationFactory({ Entity }) {
+
+    class Notification extends Entity {
+
+        //----------------------------------------------------------------------
+        // Public
+        //----------------------------------------------------------------------
+
+        /**
+         * @static
+         * @param {Object} data
+         * @return {Object}
+         */
+        static convertData(data) {
+            const data2 = {};
+            if ('id' in data) {
+                data2.id = data.id;
+            }
+            if ('notification_type' in data) {
+                data2.notification_type = data.notification_type;
+            }
+            if ('notification_status' in data) {
+                data2.notification_status = data.notification_status;
+            }
+            if ('failure_type' in data) {
+                data2.failure_type = data.failure_type;
+            }
```

Comment:
alphabetical order of `data` keys that are processed (ok for exception in partitioning conversions as attributes/relations)

Reply:
ok